.github/workflows/deploy-my-kibana.yml

```diff
@@ -17,14 +17,35 @@ on:
 
 permissions:
   contents: read
+  # if you listen for PRs, use this to use some comment reactions
+  pull-requests: write
 
 jobs:
+  trim-github-comment:
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/oblt-deploy') }}
+    runs-on: ubuntu-latest
+    outputs:
+      comment: ${{ steps.trim.outputs.result }}
+    steps:
+      - uses: elastic/oblt-actions/github/comment-reaction@v1
+        with:
+          repository: ${{ github.repository }}
+          comment-id: ${{ github.event.comment.id }}
+          github-token: ${{ github.token }}
+          emoji: 'eyes'
+
+      - id: trim
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          result-encoding: string
+          script: return `${{ github.event.comment.body }}`.trim()
```

Comment:
What do you think about putting it into an env first. I think this has a potential for script injection

Reply:
good call! 🙇

Reply:
we must avoid interpolate users inputs

Reply:
RCE

Reply:
See [`a79d431` (#205876)](https://github.com/elastic/kibana/pull/205876/commits/a79d4312bbb544cb624bb4d3089c21daa91fc157)